.meteor/versions

```diff
@@ -1,20 +1,20 @@
 accounts-base@1.4.2
 accounts-facebook@1.3.1
 accounts-oauth@1.1.15
-accounts-password@1.5.0
+accounts-password@1.5.1
 accounts-twitter@1.4.1
 accounts-ui@1.3.0
 accounts-ui-unstyled@1.4.0
 aldeed:collection2@2.10.0
 aldeed:collection2-core@1.2.0
 aldeed:schema-deny@1.1.0
 aldeed:schema-index@1.1.1
-aldeed:simple-schema@1.5.3
+aldeed:simple-schema@1.5.4
```

Comment:
Thanks @domi41 for the PR update. I see here the same issue: Many package changes that, if I'm not wrong, are not related with this PR, so we can introduce unexpected bugs. If you have some more spare time can you clean up more `.meteor/versions`?